service/awsconfig/v7/resource/lifecycle/current.go

```diff
@@ -5,5 +5,6 @@ import (
 )
 
 func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interface{}, error) {
+	r.logger.LogCtx(ctx, "level", "debug", "message", "lifecycle resource got executed")
```

Comment:
Just some debug logging to see it is executed and does things as configured by the informer.